app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -395,4 +398,13 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   def inspector_image
     'docker.io/openshift/image-inspector:2.0'
   end
+
+  def inspector_proxy_env_variables
+    settings = ext_management_system.custom_attributes
+    PROXY_ENV_VARIABLES.each_with_object([]) do |st, env|
```

Comment:
The `PROXY_ENV_VARIABLES` seem to be mutually exclusive (`no_proxy` vs. `http_proxy` vs. `https_proxy`).  Are you expecting there to be multiple custom attributes per EMS for these?

For instance, will a single EMS have both a `no_proxy` settings *and* an `https_proxy` setting?  If it does, what does that mean?

Either way, it seems better to simply query for *all* of these values, instead of *each* of these values.

```ruby
   settings.where("section = ? and name in (?)", ATTRIBUTE_SECTION, PROXY_ENV_VARIABLES)
```